tests/unit/algorithms/common/adapters/mmcv/nncf/test_helpers.py

```diff
@@ -0,0 +1,309 @@
+# Copyright (C) 2021-2023 Intel Corporation
```

Comment:
Could you add __init__ file in this folder? It would raise the pytest errors because some tasks also made a file named "test_helpers".

Reply:
added in https://github.com/openvinotoolkit/training_extensions/pull/1669/commits/a2b5d225c909115979040089cbc7fccf8b5f98ec